.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+<!-- Remove the sections that are not applicable -->
+
+## Describe your changes
+
+<!-- Descriptions can consist of describing the change and/or images of the change -->
+
+## Checklist before requesting a review
+
+- [ ] Is the PR title and first commit semantic?
```

Comment:
at minimum, the title needs to be semantic. I'm not 100% sure about the commit though.

this might not fit the format but should we link the commit guidelines? unnecessary if we're already doing that somewhere else. https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits